app/views/shop/products/_summary.html.haml

```diff
@@ -1,21 +1,18 @@
 .product-thumb
   %a{"ng-click" => "triggerProductModal()"}
-    %i.ofn-i_057-expand
-    %img{"ng-src" => "{{::product.primaryImageOrMissing}}", "ng-click" => "triggerProductModal()"}
+    %img{"ng-src" => "{{::product.primaryImageOrMissing}}"}
 
 .row.summary
-  .small-10.medium-10.large-11.columns.summary-header
+  .small-10.medium-10.large-11.summary-header
     %h3
       %a{"ng-click" => "triggerProductModal()", href: 'javascript:void(0)'}
         %span{"ng-bind" => "::product.name"}
-        %i.ofn-i_057-expand
-    %small
-      %em
-        = t :products_from
+    %p.product-description{ng: {bind: "::product.description", click: "triggerProductModal()"}}
+    .product-producer
+      = t :products_from
       %span
         %enterprise-modal
-          %i.ofn-i_036-producers
           %span{"ng-bind" => "::enterprise.name"}
-  .small-2.medium-2.large-1.columns.text-center
-    .taxon-flag
```

Comment:
this makes this line dead:
app/assets/stylesheets/darkswarm/shop.css.scss
   11: @import "shop-taxon-flag";
and the file
app/assets/stylesheets/darkswarm/_shop-taxon-flag.css.scss:

these can be deleted as well, right?